src/pybind/mgr/volumes/module.py

```diff
@@ -37,7 +37,8 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
         },
         {
             'cmd': 'fs volume rm '
-                   'name=vol_name,type=CephString',
+                   'name=vol_name,type=CephString '
+                   'name=yes_i_really_mean_it,type=CephString,req=false ',
```

Comment:
This is what I see,
```
$ ceph fs volume rm --help
fs volume rm <vol_name> {<yes_i_really_mean_it>}                                                          Delete a CephFS volume
``

It's not obvious to me that I've to pass --yes-i-really-mean-it to delete the fs volume. Please include that in the `desc` below.

Reply:
And prefer that you change the option name to `yes-i-really-mean it`. This way the help would look like,
```
fs volume rm <vol_name> {<yes-i-really-mean-it>}
```